nep-Y.mediawiki

```diff
@@ -210,11 +210,11 @@ Neo-specific codes are documented here, generic JSON-RPC errors like -32602 or
 |-
 | getapplicationlog   || -105
 |-
-| getblock            || -101
+| getblock            || -101, -109
 |-
-| getblockhash        || -109
+| getblockhash        || -101, -109
 |-
-| getblockheader      || -101
+| getblockheader      || -101, -109
```

Comment:
getblock/getblockheader are OK (commit them with "these can autoresolve height to hash, so height-related error code is applicable as well")